internal/v1/handler.go

```diff
@@ -299,6 +292,26 @@ func (h *Handlers) GetComposeStatus(ctx echo.Context, composeId uuid.UUID) error
 	return ctx.JSON(http.StatusOK, status)
 }
 
+func (h *Handlers) handleNotFoundResponse(resp *http.Response) error {
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return echo.NewHTTPError(http.StatusInternalServerError, "Failed to read response body").SetInternal(err)
+	}
+	// Composes can get deleted in composer, usually when the image is expired
```

Comment:
This comment is specific about `composes` but the function is now very generic, it can handle any NotFoundResponses and amend them with the body of the reply. So it seems to me the comment should be moved into GetComposeStatus() maybe here https://github.com/osbuild/image-builder/pull/1324/files#diff-7ca1815ab9c73f0b2b01afdd43fd976e6a77d5fcec07e7f5076251c43ff0dd4dR262 ?

Reply:
Thank you I think I should remove it